### PR TITLE
Add style guide remark about variables that hold time.

### PR DESCRIPTION
**Pull request by MrsTonedOne**

### CODE REVIEW

File: .github/guides/STYLE.md [hunk 1]
```diff
@@ -211,6 +211,9 @@ While DM allows other ways of declaring variables, this one should be used for c
 ### Use descriptive and obvious names
 Optimize for readability, not writability. While it is certainly easier to write `M` than `victim`, it will cause issues down the line for other developers to figure out what exactly your code is doing, even if you think the variable's purpose is obvious.
 
+#### Any time variable or argument that uses a unit of time other than decisecond must include the unit of time in the name.
```
Comment (san7890) [inline .github/guides/STYLE.md:214]:
```suggestion
#### Any variable or argument that uses a unit of time other than decisecond must include the unit of time in the name.
```

i feel like that first time is a bit redundant given how many other times (haha) we mention time in this title, kinda takes me out of it. were you meaning something like:

```suggestion
#### Any time a variable or argument that uses a unit of time other than decisecond must include the unit of time in the name.
```

Hard to tell

Comment (MrStonedOne) [reply .github/guides/STYLE.md:214]: it means any variable that holds time, time variable, so its still needed.

Comment (zxaber) [reply .github/guides/STYLE.md]: `All time variables or arguments that use a unit of time...` may be more clear.

File: .github/guides/STYLE.md [hunk 2]
```diff
@@ -211,6 +211,9 @@ While DM allows other ways of declaring variables, this one should be used for c
 ### Use descriptive and obvious names
 Optimize for readability, not writability. While it is certainly easier to write `M` than `victim`, it will cause issues down the line for other developers to figure out what exactly your code is doing, even if you think the variable's purpose is obvious.
 
+#### Any variable or argument that holds time and uses a unit of time other than decisecond must include the unit of time in the name.
+For example, a proc argument named `delta_time` that marks the seconds between fires could confuse somebody who assumes it stores deciseconds. Naming it `delta_time_seconds` makes this clearer, naming it `seconds_per_tick` makes its purpose even clearer.
```
Comment (LemonInTheDark) [inline .github/guides/STYLE.md:215]: I wonder how often this comes up, I don't want to bloat this if it's rare enough to not be an issue.
s a valid point tho shit sucks

**Review by san7890 — APPROVED** [review]
ultimately I am okay with this

**Comment by san7890** [comment]
@tgstation/commit-access any final thoughts/reservations/concerns/arguments regarding this addition to the style guide?

**Pull request merged**